### PR TITLE
fix: fix Alerts nested navigation issue

### PR DESCRIPTION
**Pull request by MrSltun**
This PR resolves [] 

### Description

This PR updates the navigation from the empty state of the Alerts screen in the new collector profile

### Demos

| | Before | After |
|---|---|---|
| Android | <video src="https://github.com/user-attachments/assets/d44a3e81-f32f-4cd7-b723-4b9430b91d25" /> | <video src="https://github.com/user-attachments/assets/f4f892d3-e430-4529-919b-f808b29956df" /> |
| iOS | <video src="https://github.com/user-attachments/assets/c7d80ba8-74b1-4ca6-84b3-85c5b28b776c" /> | <video src="https://github.com/user-attachments/assets/cc664cab-f659-4f5f-bb48-35f7ed521621" /> |	

### PR Checklist

- [X] I have tested my changes on **iOS** and **Android**.
- [X] I hid my changes behind a **[feature flag]**, or they don't need one.
- [X] I have included **screenshots** or **videos**, or I have not changed the UI.
- [X] I have added **tests**, or my changes don't require any.
- [X] I added an **[app state migration]**, or my changes do not require one.
- [X] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [X] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- 

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- fix navigation to home in Alerts empty state - mrsltun

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (fix navigation to home in Alerts empty state - mrsltun)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0be6f37c6701bd96fb683a9c69cc6aa32875a766
</p>

File: src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const EmptyMessage: React.FC = () => {
           <InfoSection title={t.match.title} body={t.match.body} icon={<ArtworkIcon />} />
         </Join>
         <Spacer y={4} />
-        <Button block variant="outline" onPress={() => navigate("/")}>
+        <Button block variant="outline" onPress={() => switchTab("home")}>
```
Comment (dblandin) [inline src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx:80]: **question**: I see a few other places in the codebase with `navigate("/")`. Should all of those be updated to instead switch to the home tab? Did we ever have behavior in our navigation layer to automatically switch the appropriate tab when attempting to navigate to the root route?

Comment (MrSltun) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx:80]: I'm not sure why it's happening now, it used to work normally before (tested it on 8.46) 🤷 
We do have some instances of `navigate("/")` in the code, I would probably test them as well and see

Comment (brainbicycle) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx:80]: i think instead we want to use isRootViewForTabName: "home" in AppRegistry for Home module, and it should automatically switch tabs, I am not sure if there is a reason that was removed, I think it used to be there?

Comment (brainbicycle) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx:80]: think it broke here: https://github.com/artsy/eigen/pull/10524/files cc @nickskalkin any reason we can't have that param still?

Comment (dblandin) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx:80]: Maybe an oversight but that PR (and [this specific change](https://github.com/artsy/eigen/pull/10524/files#diff-58527d675be919b77cc038d04af69bc2ef8e38ce293c03f9e7779b8efda67337R491)) shouldn't have leaked into other live screens.

So if this fixes the navigation issue I think we should incorporate it and we can fix any new home view fallout as a follow-up.

Comment (nickskalkin) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx]: @brainbicycle sorry I don't understand [this change](https://github.com/artsy/eigen/pull/10524/commits/46525d5f8d0d63438016ba84689cad48ecd6d724#diff-58527d675be919b77cc038d04af69bc2ef8e38ce293c03f9e7779b8efda67337L491). I am the author of this PR, but It was picked up by @MounirDhahri once I went to vacation, maybe he can explain once he is back?

Comment (brainbicycle) [reply src/app/Scenes/SavedSearchAlertsList/Components/EmptyMessage.tsx]: thanks! I think @dblandin is probably right it was just an oversight but we can come back to it if it was intentional

**Pull request merged**